pkgs/by-name/va/vacuum-go/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, vacuum-go }:
+
+buildGoModule rec {
+  pname = "vacuum-go";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "daveshanley";
+    repo = "vacuum";
+    # using rev because by version gives: 'the given path has multiple possibilities' error
+    rev = "4a996f877ef6738c0615d0c81b9f19dfeb248ee8";
```

Comment:
Does `rev = "refs/tags/v${version}";` work?

Reply:
ha! yes, thank you, fixed